File: 준영/프로그래머스/LV2/가장 큰 수.js
function solution(numbers) {
  var answer = ''
  var temp_numbers = numbers.map(x => String(x))
  temp_numbers.sort((a, b) => {
    var temp_a = String(a) + String(a) + String(a)
    var temp_b = String(b) + String(b) + String(b)
    if (temp_a > temp_b) {
      return -1
    }
    if (temp_a < temp_b) {
      return 1
    }
    return 0
  })
  answer = temp_numbers.join('')
  if (Number(answer) === 0) return '0'
  return answer
}


console.log(solution([6, 10, 2]))
console.log(solution([3, 30, 34, 5, 9]))

File: 준영/프로그래머스/LV2/가장 큰 수.py
def solution(numbers):
    answer = ''
    numbers = list(map(str, numbers))
    numbers.sort(reverse=True, key=lambda x: x*10)
    answer = ''.join(numbers)
    answer = str(int(answer))
    return answer

print(solution([6, 10, 2]))
print(solution([3, 30, 34, 5, 9]))